library/weno_log_sync.php

```diff
@@ -10,9 +10,6 @@
 
 use OpenEMR\Rx\Weno\Container;
 
-function log_sync()
```

Comment:
@juggernautsei , Isn't the background service calling this function? The fix here would likely be to use the function name declared in the background service entry, which is function name `start_weno()`

Reply:
I am scratching my head, I think you are right, I got turned around. Let me fix that.